docs/docsite/rst/porting_guides/porting_guide_core_2.19.rst

```diff
@@ -650,6 +650,18 @@ Values formerly represented by that type will now appear as a tagged ``str`` ins
 Special handling in plugins is no longer required to access the contents of these values.
 
 
+No implicit conversion of non-string keys in returned dictionaries
+------------------------------------------------------------------
+
+Previously, ``ansible-core`` relied on the JSON dumping to implicitly convert ``non-string dictionary keys`` to strings when modules returned data.
+
+Starting with this release, modules must explicitly convert any non-string keys to strings before passing dictionaries to the ``AnsibleModule.exit_json()`` method of ``ansible-core``.
```

Comment:
Are there any specific docs on how to implement the conversion? It would probably be good to link that here.

Reply:
SGTM, added one, PTAL

Reply:
@oraNod added more, PTAL